socialhome/streams/views.py

```diff
@@ -10,12 +12,67 @@ class BaseStreamView(ListView):
     ordering = "-created"
     paginate_by = 30
     stream_name = ""
+    vue = False
+    queryset = []
+
+    def dispatch(self, request, *args, **kwargs):
+        self.vue = bool(request.GET.get("vue", False))
+        return super().dispatch(request, *args, **kwargs)
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context["stream_name"] = self.stream_name
+        if self.vue:
+            context["json_context"] = self._define_context_json(self.stream_name)
         return context
 
+    def _define_context_json(self, stream_name):
```

Comment:
Please add a  `# pragma: no cover` since this is temporary code and we are not going to have tests for it and so coverage shouldn't be affected. See https://coverage.readthedocs.io/en/coverage-4.4.1/excluding.html